### ISSUE TITLE
Release a new major version after PR #349

### ISSUE THREAD

**Issue by Asvarox** [luggit/react-native-config#380]
Seems like that #349 considerably changed the way to use .env values in iOS build settings/info.plist file. It might be a good idea to release the version of the lib with these changes, as installation instructions in current `readme.md` won't work for the most recently published NPM package.

**Comment by bericp1** [comment]
As a stop gap measure, I've installed the specific commit on `master` right now (`89a602bf8be3808838403a97afaf915caeec76c2`) directly from GitHub and it's working on both Android and iOS. Tested the `Info.plist` iOS modifications and those are also working on `master`.

Obviously installing directly from GitHub isn't recommended but as a temporary measure, anyone who needs these latest changes can run:

```
yarn add react-native-config@"luggit/react-native-config#89a602bf8be3808838403a97afaf915caeec76c2"
```

**Comment by erwann-miriel-nv** [comment]
It would really help to have a new release. I got stuck for a day trying to solve the "'GeneratedDotEnv.m' file not found" until I installed from github

**Comment by Sbphillips19** [comment]
@erwann-miriel-nv add in the script found here-  https://github.com/luggit/react-native-config/issues/365

I forgot to do pod install and ended up back here, sure it will happen again with this library.

**Comment by MrLoh** [comment]
@pedro @jordanbrown @erickreutz can someone please make an npm release or fork this package, what's the point of merging PRs and then not releasing an update?

**Comment by pedro** [comment]
Oops, sorry for the delay folks, `0.12` has been released with #349.
For updates on repo ownership and updates follow #382.
Thanks!
